editPage.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Edit Profile</title>
+    <link rel="stylesheet" href="./editPage.css">
+  </head>
+  <body>
+    <div class="nav-div">
+      <div class="nav_left">
+        <a href="./index.html">
+          <img
+            class="nav_logo"
+            src="./assets/Icons/Layer 3.svg"
+            alt="Credard logo"
+        /></a>
+
+        <nav class="nav_menu">
+          <ul class="nav_list">
+            <li>
+              <a class="nav_link" href="./how-it-works.html">How it works</a>
+            </li>
+            <li><a class="nav_link" href="#">Library</a></li>
+            <li><a class="nav_link" href="./team.html">The team</a></li>
+            <li><a class="nav_link" href="./contactUs.html">Contact us</a></li>
+          </ul>
+        </nav>
+      </div>
+      <div class="nav_right">
+        <a class="nav_login" href="./login.html">Login</a>
+        <a class="nav_register" href="./register.html">Register</a>
+      </div>
+      <button id="hamburger-btn" class="nav_btn">
+        <img src="./assets/Icons/menu - ham.svg" alt="" />
+      </button>
+    </div>
+    <div class="nav_hamburger-menu">
+      <nav id="hamburger-menu" class="ham_menu hide">
+        <ul>
+          <li>
+            <a class="ham_link" href="./how-it-works.html">How it works</a>
+          </li>
+          <li><a class="ham_link" href="#">Library</a></li>
+          <li>
+            <a class="ham_link" href="./team.html">The team</a>
+          </li>
+          <li><a class="ham_link" href="./contactUs.html">Contact us</a></li>
+        </ul>
+        <div class="ham_btns">
+          <a class="ham_login" href="./login.html">Login</a>
+          <a class="ham_register" href="./register.html">Register</a>
+        </div>
+      </nav>
+    </div>
+
+    <main class="edit">
+        <div class="edit_bar">
+            <div class="edit_profileImage">
+                <img class="image" src="./assets/images/profilePicture.svg" alt="Profile Picture">
+                <button><a href="">Edit image &nbsp; <img src="./assets/Icons/edit-pen.svg" alt=""></a></button>
+            </div>
+        </div>
+
+        <form action="">
+            <div class="edit_username">
+                <label for="username">Change Username</label><br>
+                <input type="text" id="username">
+            </div>
+
+            <P>Change your password</P>
```

Comment:
This is supposed to look like this 
![image](https://user-images.githubusercontent.com/60205816/183939163-f5fbfca1-5b0c-4083-a1de-47a68bbd449e.png)


But this is what we have 
![image](https://user-images.githubusercontent.com/60205816/183939365-2d6f4974-c9af-4bf4-94f7-f7295f8297c9.png)